packages/ui/dialogs/my-account/index.tsx

```diff
@@ -478,7 +475,7 @@ class MyAccount extends Component<Props, State> {
                 </View>
               </TouchableHighlight>
               {showNicknameInput ? <View style={style.spaceHorizontalM}>
-                <Text style={[style.text, style.spaceTopM, style.center]}>{setNickname}</Text>
+                {nickname ? <Text style={[style.text, style.spaceTopM, style.center]}>{setNickname}</Text> : null}
```

Comment:
We no longer need the localizations for 'changeProfilePic' and 'addProfilePic' in the various localization files... not sure if you can pull out the original commit or not.

Reply:
![image](https://user-images.githubusercontent.com/1081476/44360894-17103d00-a471-11e8-8b65-f85eeab5f852.png)

Camera icon is centered, which doesn't look good on our default avatar. It either needs a gray background or it could be placed top-center on the Image.

Reply:
good call, I'll fix that camera image